spec/helpers/moods_helper_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+describe MoodsHelper do
+  subject { TestClass.new(build(:mood, user_id: 999, id: 999)) }
+
+  class TestClass
```

Comment:
I think we can move the initialisation to a before block and do away with the testClass

eg 
```ruby
before do
  @mood = mood
end
```

And tests will be in the format

```ruby
describe '#new_mood_props' do
    it 'calls method with correct params' do
      allow(FormHelper).to receive(:new_form_props).with(mood_form_inputs_res, '/moods').and_return(":new_form_props called")
      expect(FormHelper).to receive(:new_form_props).with(mood_form_inputs_res, '/moods')
      expect(new_mood_props).to eq ":new_form_props called"
    end
  end